dev/com.ibm.ws.logging_fat/fat/src/com/ibm/ws/logging/fat/JSONFieldsTest.java

```diff
@@ -309,6 +310,19 @@ public void testRenameAndOmitExtFields() throws Exception {
         getHttpServlet("/extFields/CreateLogs", server_xml); //see omitted field
         assertNull("The extension field name was not omitted", server_xml.waitForStringInLogUsingMark("ext_testExtension"));
 
+        server_xml.setMarkToEndOfLog();
+
+        //rename extension fields
+        setServerConfiguration(true, "ext_testExtension:RENAME", server_xml);
+        getHttpServlet("/extFields/CreateLogs", server_xml); //see new renamed field
+        assertNotNull("The extension field name was not renamed", server_xml.waitForStringInLogUsingMark("RENAME"));
+
+        //remove/unregister extension field
+        getHttpServlet("/extFields/removeExtFields", server_xml);
+        server_xml.setMarkToEndOfLog();
+        getHttpServlet("/extFields/CreateLogs", server_xml);
+        assertNull("The extension field name was not unregistered", server_xml.waitForStringInLogUsingMark("ext_testExtension"));
```

Comment:
I think here you would want to watch for "RENAME" again since that's the last thing the test configured the field to be named.